README.md

```diff
@@ -1,41 +1,8 @@
 # web-scraper-helper
 A tool to test web scraping rules.
```

Comment:
It doesn't have to be in this PR, but we will want to better describe this project in this readme.
Think of someone who stumble upon this page, he should be able to understand this is related to Coveo Web Sources & what is web scraping in general. 

And a one liner about each "version" of this tool (TM & extension) linking to their readme.